python/tank/authentication/shotgun_shared/saml2_sso.py

```diff
@@ -248,20 +272,22 @@ def start_sso_renewal(self):
 
             # For debugging purposes
             # @TODO: Find a better way than to use this EV
-            if "SAML2_SHOTGUN_USING_VM" in os.environ:
-                interval = 5000
+            if "SHOTGUN_SSO_DEVELOPER_ENABLED" in os.environ:
```

Comment:
The value in itself is not that important... I set it to 5 seconds so that the SSO claims refresh are done very quickly. (These resfresh usually occur every 4 mins 20 secs, which is a bit longuish when debugging.)

So for the time being I would leave it the way it is.